### PR TITLE
feat(kfp): Update KFP pipelines

### PR DESCRIPTION
**Pull request by sylus**

### CODE REVIEW

File: docker-bits/3_Kubeflow.Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-      'kfp==1.0.0' \
-      'kfp-server-api==1.0.0' \
-      'kfp-tekton==0.3.0' \
-      'kubeflow-fairing==1.0.1' \
-      'ml-metadata==0.24.0' \
+      'kfp==1.2.0' \
+      'kfp-server-api==1.2.0' \
+      'kfp-tekton==0.5.0' \
+      'kubeflow-fairing==1.0.2' \
+      'ml-metadata==0.26.0' \
```
Comment (ca-scribner) [inline docker-bits/3_Kubeflow.Dockerfile:6]: Kf metadata requires 0.21.1 but our prev build was past that too. Just noting here to give someone a hint if it breaks

**Review by ca-scribner — APPROVED** [review]
Pytorch fails because an unpinned version somewhere in that stack changed a few days ago. Will assigned as a separate bug fix

**Comment by sylus** [comment]
@ca-scribner able to do a P.R. for the broken pytorch?

**Comment by sylus** [comment]
I'll merge this for now since master is broken already.

**Comment by ca-scribner** [comment]
Yeah I’ll either find someone to fix pytorxh today or tackle it myself

**Pull request merged**